components/datepicker/theme.css

```diff
@@ -344,7 +344,7 @@
   left: 0;
   margin-top: 16px;
   position: absolute;
-  z-index: 1;
+  z-index: 400;
```

Comment:
Just a quick question: do you have a system to define z-indexes? Am interested in that :p

Reply:
Yes we have, Stephen made these [visual guidelines](https://app.zeplin.io/project/599af68db2f4978a56dec839/screen/5bd03326767a220982b415f8).

Reply:
Stephen is still working this out, you can already find an idea of what the system should look like on Zeplin I think. I see he started it here: https://guidelines.teamleader.design/primitives/layers but there'll be more available soon